### DISCUSSION TITLE
Calibration

### DISCUSSION THREAD

**Discussion by Richlang6371** [community/community#37732] [Mobile]
On my Galaxy s21 5g I am unable to calibrate, Sky Map app, the compass. The screenshot the diagnostics page appears, to me, to be set up correctly but still not calibrated. Please help me.

**Comment by ossanna16** [comment]
Hi @Richlang6371,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Samsung Galaxy phones. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
